fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/JDBCConstants.java

```diff
@@ -48,6 +48,8 @@
     public static final String IS_DELETED = "IS_DELETED";
 
     // Special parameter names
+    public static final String PARAM_NAME_ID = "_id";
+    public static final String PARAM_NAME_LAST_UPDATED = "_lastUpdated";
     public static final String PARAM_NAME_PROFILE = "_profile";
     public static final String PARAM_NAME_TAG = "_tag";
```

Comment:
wondering if these should be in `fhir-persistence` since they aren't specific to our JDBC impl.  We probably have them already in some more common place as well (just a guess)?

Reply:
I removed them from fhir-persistence. There are existing ones in SearchConstants, so added the new ones there as well.